StartupSession/Dyalog/Utils.apln

```diff
@@ -49,8 +49,6 @@
       r←⊢/∊'V'⎕VFI⊃⎕SRC ⎕THIS
     ∇
 
-    where←{⍵/⍳⍴⍵}        ⍝ Indices of 1's in ⎕IO 1
```

Comment:
This is a utility namespace; other people's code can rely on the functions here, so we cannot simply remove them. (That said, this can pf course be `where←⍸`.)